rbac_api_with_domains.go

```diff
@@ -137,3 +137,8 @@ func (e *Enforcer) DeleteDomains(domains ...string) (bool, error) {
 	}
 	return true, nil
 }
+
+// GetAllDomains would get all domains.
+func (e *Enforcer) GetAllDomains() ([]string, error) {
+	return e.model["p"]["p"].RM.GetAllDomains()
```

Comment:
We should use the `RM` in `g` section and `gX`  (like `g`, `g2`., `g3`..)

Reply:
Hi @sagilio, I changed the "p" section to "g", but can you please mention why we did that or can give me a resource to read about that?

Reply:
@BudhirajaMadhav see: https://casbin.org/docs/en/role-managers